test/suites/dev/test-democracy/test-democracy-proposal-seconding.ts

```diff
@@ -57,7 +57,7 @@ describeSuite({
     it({
       id: "T03",
       title: "should end-up in a valid referendum",
-      timeout: 1000000,
+      timeout: 120_000,
```

Comment:
Why so long? (yes, it's shorter than 1M...)

Reply:
@crystalin can you answer this, please?

Reply:
Because it has to create 7200 blocks, which on the CI is not as fast as on my computer :)